src/main/scala/com.github.mgifos.workouts/Main.scala

```diff
@@ -93,6 +94,14 @@ object Main extends App {
             failure("Either start or end date must be entered!")
           else success))
 
+    note("")
+
+    cmd("clean").
```

Comment:
There's no need for introducing `clean` command since there's already `delete` parameter available (I would rather introduce optional mode in config to handle this situation instead of having import as default mode)